python-sdk/src/langwatch/utils/initialization.py

```
@@ -34,7 +34,7 @@ def setup(
     base_attributes: Optional[BaseAttributes] = None,
     tracer_provider: Optional[TracerProvider] = None,
     instrumentors: Optional[Sequence[Instrumentor]] = None,
-    span_exporter_exclude_rules: Optional[List[SpanExporterExcludeRule]] = [],
+    span_exporter_exclude_rules: Optional[List[SpanProcessingExcludeRule]] = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)